react.json

```diff
@@ -31,31 +31,32 @@
     "no-empty": 0,
     "no-restricted-syntax": 0,
     "no-await-in-loop": 0,
+    "no-param-reassign": 0,
+    "no-confusing-arrow": 0,
+    "no-nested-ternary": 0,
     "camelcase": 0,
     "arrow-parens": [
       "error",
       "as-needed"
     ],
-    "no-confusing-arrow": [
```

Comment:
enabling `allowParens` conflicts with prettier, which would try to remove the extra parens.

this is basically idiomatic in react code so i dont think its a big deal to disable